### DISCUSSION TITLE
Shipping Rates and Rate Breaks by Weight

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [lunarphp/lunar#2006] [Feature Requests]
When defining a Shipping Method, the `Charge By` property signifies whether the Shipping rates should be identified by Cart Total or Weight. But when defining rates, there isn't currently a way to define shipping rates by weight. 

![Arc 2024-11-01 14 09 55](https://github.com/user-attachments/assets/5ff066aa-f540-428a-bb51-0550206038b7)

![CleanShot 2024-11-01 at 14 22 51](https://github.com/user-attachments/assets/2c15c1ee-eeac-47e8-b77e-d3d2862b571a)

If a Shipping method that supports rate calculation by weight is selected, then rate breaks should allow defining pricing based on weights.

**Comment by binmosa** [comment]
This is so requested feature hope to be implemented.
